presto-main/src/main/java/com/facebook/presto/sql/analyzer/Scope.java

```diff
@@ -89,6 +89,11 @@ private Scope(
         return Optional.empty();
     }
 
+    public Optional<Scope> getParent()
```

Comment:
I don't think it's needed, see my comment on: I don't think it's needed, see my comment on https://github.com/prestodb/presto/pull/10375/files/098ddf08b4bfb2061fc3d855a307f5d14098ff5b..9949b16ce62d2a0c9b09291e3a2546cb3b01ef6b#diff-de0c3bd1446c0ac0bf16a9e1c91b20f5L403